netbox/templates/inc/table_controls_htmx.html

```diff
@@ -5,7 +5,7 @@
   <div class="col-auto d-print-none">
     <div class="input-group input-group-flat me-2 quicksearch">
       <input type="search" results="5" name="q" id="quicksearch" class="form-control px-2 py-1" placeholder="Quick search"
-        hx-get="{{ request.full_path }}" hx-target="#object_list" hx-trigger="keyup changed delay:500ms, search" />
+        hx-get="" hx-target="#object_list" hx-trigger="keyup changed delay:500ms, search" />
```

Comment:
Removed full path, as this will allow taking int account filters on the form (to facilitate pushing of the filters from the column as well).

Should not impact functionality

Reply:
> Should not impact functionality

Do we know this for sure? IIRC we specify the full path for a reason, I just can't recall why.

Reply:
From my testing, I haven't found any issues with it.